src/JitInterface/src/CorInfoHelpFunc.cs

```diff
@@ -73,9 +73,9 @@ public enum CorInfoHelpFunc
         CORINFO_HELP_NEWARR_1_ALIGN8,   // like VC, but aligns the array start
 
         CORINFO_HELP_STRCNS,            // create a new string literal
-    #if !RYUJIT_CTPBUILD
+#if !RYUJIT_CTPBUILD
```

Comment:
I made a mistake when removing the #if !RYUJIT_CTPBUILD which broke the build. My last push fixed that.